pkg/logger/logger.go

```diff
@@ -16,16 +16,23 @@ const (
 )
 
 var (
-	l    logr.Logger = logr.Discard()
-	once sync.Once
+	l              logr.Logger = logr.Discard()
+	once           sync.Once
+	outputFilename string
 )
 
-func set(logger logr.Logger) {
+func set(logger logr.Logger, out string) {
 	once.Do(func() {
 		l = logger
+		outputFilename = out
 	})
 }
 
+// GetOutputFileName returns the current outputFileName.
```

Comment:
```suggestion
// GetOutputFileName returns the path to the file where high verbosity logs are written to.
```

Or or something else, but this current comment doesn't tell me anything.